examples/seismic/tutorials/16_variable_z_topography.ipynb

```diff
@@ -0,0 +1,453 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    pdx2 = p.dx2+p.dx.dzeta*zeta_dx+(p.dx.dzeta+p.dzeta2*zeta_dx)*zeta_dx+p.dzeta*zeta_dxx
</pre><p>Would be better to use a <code>.subs()</code> here and substitute into the standard equation (better for complex transforms) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>